targets/TARGET_STM/TARGET_STM32L1/TARGET_MTB_RAK811/PinNames.h

```diff
@@ -0,0 +1,147 @@
+ *
+ * 1. Redistributions of source code must retain the above copyright notice,
+ *    this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ * 3. Neither the name of STMicroelectronics nor the names of its contributors
+ *    may be used to endorse or promote products derived from this software
+ *    without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *******************************************************************************
+ */
+#ifndef MBED_PINNAMES_H
+#define MBED_PINNAMES_H
+
+#include "cmsis.h"
+#include "PinNamesTypes.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef enum {
+    PA_0  = 0x00,
+    PA_1  = 0x01,
+    PA_2  = 0x02,
+    PA_3  = 0x03,
+    PA_4  = 0x04,
+    PA_5  = 0x05,
+    PA_6  = 0x06,
+    PA_7  = 0x07,
+    PA_8  = 0x08,
+    PA_9  = 0x09,
+    PA_10 = 0x0A,
+    PA_11 = 0x0B,
+    PA_12 = 0x0C,
+    PA_13 = 0x0D,
+    PA_14 = 0x0E,
+    PA_15 = 0x0F,
+
+    PB_0  = 0x10,
+    PB_1  = 0x11,
+    PB_2  = 0x12, // not used
+    PB_3  = 0x13,
+    PB_4  = 0x14,
+    PB_5  = 0x15,
+    PB_6  = 0x16,
+    PB_7  = 0x17,
+    PB_8  = 0x18,
+    PB_9  = 0x19,
+    PB_10 = 0x1A,
+    PB_11 = 0x1B,
+    PB_12 = 0x1C,
+    PB_13 = 0x1D,
+    PB_14 = 0x1E,
+    PB_15 = 0x1F,
+
+    PC_13 = 0x2D,
+    PC_14 = 0x2E, // not used ?
+    PC_15 = 0x2F, // not used ?
+
+    PH_0  = 0x70,
+    PH_1  = 0x71, // not used ?
+
+    // ADC internal channels
+    ADC_TEMP = 0xF0,
+    ADC_VREF = 0xF1,
+
+    ADC1_0   = PB_14,
+    ADC1_1   = PB_15,
+    ADC1_2   = PB_12,
+
+    UARTn_RX = PA_10,
+    UARTn_TX = PA_9,
+
+    SPI_RF_MOSI = PA_7,
+    SPI_RF_MISO = PA_6,
+    SPI_RF_SCK  = PA_5,
+    SPI_RF_CS   = PB_0,
+    SPI_RF_RESET= PB_13,
+
+    DIO0        = PA_11,
+    DIO1        = PB_1,
+    DIO2        = PA_3,
+    DIO3        = PH_0,
+    DIO4        = PC_13,
+    //DIO5        = ,
+
+    ANT_CTX_PA  = PA_4,
+    ANT_CRX_RX  = PB_6,
+    ANT_CBT_HF  = PB_7,
```

Comment:
Is ANT_CBT_HF a lora pin? Instead Lora needs:
RF_TCXO_EN  = PH_1,

Reply:
I have CBT_HF from : https://github.com/RAKWireless/RAK811_BreakBoard/blob/1d5483aef2c691640db747fb230ceeb96acb378b/src/boards/RAK811BreakBoard/board.h#L96
However I have no idea about what those pins are as I don't know (yet) lora at all.